src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -61,8 +61,9 @@ export class CreateScenariosComponent implements OnInit {
 
   // this value gets updated once we load the scenario result.
   scenarioState: ScenarioResultStatus = 'NOT_STARTED';
-
+  scenario: Scenario | null = null;
   scenarioResults: ScenarioResult | null = null;
```

Comment:
Are we keeping `scenarioResults` as well as `scenario` ?

Reply:
Removed scenario